meson.build

```diff
@@ -43,6 +43,14 @@ add_project_arguments(
     language: ['vala']
 )
 
+if get_option('gtk')
+    add_project_arguments(
+        '-D',
+        'INCLUDE_GTK',
```

Comment:
Same here. Just so someone doesn't look at this and say "What why wouldn't we compile Granite with GTK?" :sweat_smile: